ci/infrastructure/scripts/infrastructure.sh

```
@@ -1124,6 +1126,22 @@ createBuildReport() {
     echo "" >> $VS_MAIL_NOTIFY_BUILD_MESSAGE
     echo "" >> $VS_MAIL_NOTIFY_BUILD_MESSAGE
   fi
+  # quick and dirty conversion of the email message to an "HTML" file that'll play nice with the Jenkins HTML Publisher
+  if [ -e "$VS_MAIL_NOTIFY_BUILD_MESSAGE" ]; then
+    echo "$(eval $VS_LOG_DATESTAMP) INFO  [$VS_SCRIPTNAME] writing build report to $VS_HTML_PUBLISHER_REPORT_DIR/$VS_HTML_PUBLISHER_REPORT_FILE"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)